packages/discovery-react-components/src/components/ResultsPagination/ResultsPagination.tsx

```diff
@@ -34,6 +50,10 @@ export interface ResultsPaginationProps {
    * custom handler invoked when any input element changes in the ResultsPagination component
    */
   onChange?: (e: ResultsPaginationEvent) => void;
+  /**
+   * ref passed in by parent that can be used to reset the pagination component back to page 1
+   */
+  resetRef?: ForwardedRef<any>;
```

Comment:
```suggestion
  /**
   * Reference to imperative API
   */
  ref?: ForwardedRef<ResultsPaginationAPI>;
```

And define the type `ResultsPaginationAPI`. Oh, and export it from components (since it'll be used by consumers).